ccd-definition/CaseField.json

```diff
@@ -136,5 +136,23 @@
     "Label": "Response deadline",
     "FieldType": "DateTime",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "systemGeneratedCaseDocuments",
+    "Label": "System generated Case Documents",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "CaseDocument",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "id",
```

Comment:
@dharmendrak - what's the id for? How does this differ from the CASE_REFERENCE and how will this be used? id is really vague from a modelling standpoint so would like to be clear in what this is.

Reply:
this is CCD ID (ccd data store id) -> can be reworded as case Identifier

Reply:
it's removed